frappe/desk/page/setup_wizard/setup_wizard.py

```diff
@@ -405,6 +404,7 @@ def make_records(records, debug=False):
 
 		try:
 			doc.insert(ignore_permissions=True)
+			frappe.db.commit()
```

Comment:
If there's a commit here, there should also be a rollback to revert every errored Document creation

Reply:
👍